API.js

```diff
@@ -0,0 +1,104 @@
+
+let quote ={text: ''}
+let mainUser = {}
+let friendsList = {friends: []}
+let pokemon = {}
+let about = {meat: ''}
+
+class API {
```

Comment:
The way you structured your classes here is really nice. I'm not sure I would do this for such a small project but in general, it is good practice to separate your services which fetch different data. Nice job.